src/Custom/Audio/AudioClient.cs

```diff
@@ -229,6 +229,11 @@ public virtual AsyncCollectionResult<StreamingAudioTranscriptionUpdate> Transcri
         Argument.AssertNotNull(audio, nameof(audio));
         Argument.AssertNotNullOrEmpty(audioFilename, nameof(audioFilename));
 
+        if (string.Equals(_model, "whisper-1", StringComparison.OrdinalIgnoreCase))
```

Comment:
Should we consider extracting that into a method so that the message and model aren't hard-coded in a few places?

Reply:
Thanks for the suggestions! I've updated the code to extract the check into a private method.